scripts/genie.lua

```diff
@@ -1237,14 +1227,18 @@ configuration { "vs20*" }
 			"userenv",
 		}
 
+if _OPTIONS["vs"]==nil then
 		buildoptions {
-			"/Zc:preprocessor",
-			"/utf-8",
+			"/bigobj",
 			"/permissive-",
-			"/w45038", -- warning C5038: data member 'member1' will be initialized after data member 'member2'
+			"/utf-8",
+			"/Zc:enumTypes",
+			"/Zc:preprocessor",
+			"/Zc:templateScope",
 		}
 
 		buildoptions {
+			"/w45038", -- warning C5038: data member 'member1' will be initialized after data member 'member2'
```

Comment:
Where’s this actually being hit?  We have this error enabled for other compilers, so it shouldn’t be happening in our own code.

Reply:
This is actually enabling the warning, not disabling it; I don't know why Microsoft made it off by default.